CMakeLists.txt

```diff
@@ -4,7 +4,13 @@
 #
 cmake_minimum_required(VERSION 3.17)
 
+option(BUILD_FOR_PICO2 "Build firmware for Pico 2")
+if(BUILD_FOR_PICO2)
+set(BOARD "raspberry_pi_pico2")
```

Comment:
why don't you just allow(or even require) to set BOARD directly? Otherwise this gets more complicated with every new board.

Reply:
Well, why not :) Using `BOARD` now.